src/Unity.Mathematics.Shaders/hlsl.cs

```diff
@@ -2,105 +2,367 @@
 
 namespace Unity.Mathematics.Shaders
 {
+    /// <summary>
```

Comment:
Is filtering a valid way to avoid documenting these?

I don't actually want to generate any documentation for these because these functions aren't implemented and I think they need to be deleted outright, but I'm not yet ready to delete them.

Reply:
Yes, filtering them out is appropriate, then.

Reply:
Use the following to supress the entire Shaders namespace:

`  - exclude:
      uidRegex: Unity\.Mathematics\.Shaders$
      type: Namespace
`